packages/db/src/translate/executions.ts

```diff
@@ -0,0 +1,88 @@
+import Knex from 'knex';
+
+import { RecordDoesNotExist } from '@cumulus/errors';
+import { ExecutionRecord } from '@cumulus/types/api/executions';
+import Logger from '@cumulus/logger';
+import { PostgresExecution } from '../types/execution';
+import { ExecutionPgModel } from '../models/execution';
+import { CollectionPgModel } from '../models/collection';
+import { AsyncOperationPgModel } from '../models/async_operation';
+
+/**
+ * Translate execution record from Dynamo to RDS.
+ *
+ * @param {AWS.DynamoDB.DocumentClient.AttributeMap} dynamoRecord
+ *   Source record from DynamoDB
+ * @param {AWS.DynamoDB.DocumentClient.AttributeMap} knex
+ *   Knex client
+ * @returns {PostgresExecutionRecord} - converted Execution
+ */
+export const translateApiExecutionToPostgresExecution = async (
+  dynamoRecord: ExecutionRecord,
+  knex: Knex
+): Promise<PostgresExecution> => {
+  const execution = new ExecutionPgModel();
+  const collection = new CollectionPgModel();
+  const asyncOperation = new AsyncOperationPgModel();
```

Comment:
Nit: I'd prefer these be named with a suffix of `PgModel` for clarity

Reply:
Yup, absolutely. Hopefully in the main app logic we can avoid naming things like this and just have stuff like `ExecutionModel`. Here though, the whole point is to handle both types so yeah, totally good with anything that clears confusion 😄